buildSrc/subprojects/plugins/src/main/kotlin/org/gradle/plugins/lifecycle/LifecyclePlugin.kt

```diff
@@ -171,7 +171,7 @@ class LifecyclePlugin : Plugin<Project> {
             description = "Build production distros and smoke test them"
             group = "build"
             dependsOn(":distributions:verifyIsProductionBuildEnvironment", ":distributions:buildDists",
-                ":distributions:integTest", ":docs:check", ":docs:checkSamples")
```

Comment:
Wait a sec, does `docs:check` only depend on `docs:docsTest`? Is there any other tasks `check` depends on?

Reply:
You are right, there are missing tasks. Let me look them up.

Reply:
I've checked the task dependencies and only one task was missing: `releaseNotes`. I've restored it.